src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,38 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post: {
+  title,
+  body,
+  comments,
+  user,
+} }) => (
+  <li className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {title}
+      </h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={user} />
```

Comment:
```suggestion
        {user && <UserInfo user={user} />}
```
technically user can be null and UserInfo component can be failed.
![Screenshot from 2023-12-01 13-54-01](https://github.com/mate-academy/react_static-list-of-posts-js/assets/33029864/77205790-8c5a-46a5-9602-6266b6fe0f04)

suggest to render component UserInfo conditionally and just in case when user exists.